src/applications/pensions/containers/ConfirmationPage.jsx

```diff
@@ -33,77 +27,124 @@ class ConfirmationPage extends React.Component {
 
     return (
       <div>
-        <h3 className="confirmation-page-title">Claim submitted</h3>
-        <p>
-          We process claims in the order we receive them. Please print this page
-          for your records.
-        </p>
-        <p>We may contact you for more information or documents.</p>
+        <h2>Your Veteran's Pension application</h2>
+
+        <va-alert status="success">
+          <h3>Thank you for submitting your Veterans Pension application.</h3>
+          <p>
+            We've received your Veterans Pension application (VA Form
+            21P-527EZ). After we complete our review, we'll mail you a decision
+            letter with the details of our decision.
+          </p>
+        </va-alert>
+
         <div className="inset">
```

Comment:
We should be able to use the `va-summary-box` in 'vets-website' now
```suggestion
        <va-summary-box uswds>
          <h3 slot="headline" className="vads-u-margin-top--0">
            Your information for this application
          </h3>
```